client/packages/common/src/ui/components/inputs/TextInput/BasicTextInput.tsx

```diff
@@ -2,20 +2,24 @@ import React, { FC } from 'react';
 import {
   Box,
   StandardTextFieldProps,
+  SxProps,
   TextField,
+  Theme,
   Typography,
 } from '@mui/material';
 
-export type BasicTextInputProps = StandardTextFieldProps;
+export interface BasicTextInputProps extends StandardTextFieldProps {
+  boxSx?: SxProps<Theme>;
```

Comment:
there is an `sx` a `boxSx` and and `InputProps.sx` now. For a basic text input this is a bit confusing to me :) At least we should document more clearly what they do. Can sx and boxSx somehow merged? @mark-prins maybe you have a better idea?